webpack/assets/javascripts/react_app/components/common/forms/DateTime/DateTime.js

```diff
@@ -35,8 +36,8 @@ const DateTime = ({
       }
     >
       <DateTimePicker
-        hiddenValue={isRequired}
-        value={value}
+        hiddenValue={isBlank(value) && !isRequired}
+        value={!isBlank(value) ? value : new Date}
```

Comment:
it's a bit unclear, and because the dateTime component still under development in patterfnly, I prefer a smaller change for it, I've just added a prop `hideValue` 

```js
      <DateTimePicker
        hiddenValue={**hideValue** || !isRequired}
        value={value}
        id={`template-date-input-${id}`}
        inputProps={{
          name: `${template}[input_values][${id}][value]`,
        }}
        locale={currentLocale}
      />
```